generator/internal/codegen_utils.cc

```diff
@@ -140,6 +140,26 @@ ProcessCommandLineArgs(std::string const& parameters) {
   return command_line_args;
 }
 
+std::string CopyrightLicenseFileHeader() {
+  static std::string const kHeader =  // clang-format off
+  "// Copyright 2020 Google LLC\n"
```

Comment:
I'm not certain, but I think we may want the copyright year to be the current year. FWIW, you can get the current year with

```cc
#include "absl/time/time.h"
...
std::string year_string = absl::FormatTime("%Y", absl::Now(), absl::UTCTimeZone());
```

Reply:
AFAIK the copyright notice should include the year the file was created, and should not change each time we regenerate the file.  Not sure how that will work with our plans to always regenerate these files and to keep them committed.

Reply:
Are you thinking that this tool might be used by third-parties with their own code?  If so, the copyright notice is not appropriate.  Maybe make it optional?

Reply:
I'm not thinking about anyone else using this tool. But if next year we start generating files for a new service, 2020 will be the wrong year.